superset-frontend/src/dashboard/components/nativeFilters/FilterBar/FilterControls/FilterValue.tsx

```diff
@@ -322,7 +323,7 @@ const FilterValue: React.FC<FilterControlProps> = ({
       ) : (
         <SuperChart
           height={HEIGHT}
-          width="100%"
+          width={RESPONSIVE_WIDTH}
```

Comment:
What's the advantage of using this `RESPONSIVE_WIDTH` instead of `100%`?

Reply:
The RESPONSE_WIDTH change is related to the topic no.2 (`2. Redundant filter re-renders due to the dynamic sizing`)